src/mixins/labels.js

```diff
@@ -16,10 +16,31 @@ export default {
       }
 
       if (c.type === 'group') {
-        return this.$store.getters['channels/otherMembersOf'](c.channelID, this.$auth.user.userID)
+        let groupMembers = this.$store.getters['channels/otherMembersOf'](c.channelID, this.$auth.user.userID)
```

Comment:
Let's extract all this logic into new component - `<group-members>` that will have that single responsibility -- display members of a group. Be smart with props.